Support double height plants

aedans opened the pull request:
Fixes #42 for 1.16, not sure if it works for older versions.

aedans committed:
Support double height plants

Snownee commented:
Thanks for contributing! There are some small issues:
1. Right clicking double grass's upper state with snow breaks double grass
2. Breaking double grass's upper state, the lower state will stay

aedans committed:
Fix placing snow on upper state breaking double grass
Fix breaking upper state in survival keeping lower state
Fix placing block on upper state keeping lower state
Allow stacked snow tile blocks to fall

aedans commented:
Issues have been mostly fixed. There's still a minor bug where placing a block on the upper state of tall grass on snow won't play the breaking effect, but I'm not sure how to fix it.

Snownee commented:
Thank you very much!
